htdocs/installdb.php

```diff
@@ -51,6 +51,7 @@
 // smarty to use when it gets autoloaded.
 require_once __DIR__ . "/../php/installer/Database.class.inc";
 require_once __DIR__ . "/../php/installer/NDB_Config.class.inc";
+use \LORIS\Installer\Database as Database;
```

Comment:
How do this distinguish between Database (\Database) and Database (\LORIS\Installer\Database) ?

Reply:
But if this file is not in a namespace, then it is in the '\' by default. The same '\' as in \Database

Reply:
the use alias will take precedence (and only the installer version is included, the installdb.php doesn't autoload)